src/Client.ts

```diff
@@ -4,7 +4,7 @@ import {
   GatewayOpcodes
 } from 'discord-api-types/v9'
 import * as D from 'discord.js'
-import WebSocketShard from '../node_modules/discord.js/src/client/websocket/WebSocketShard'
+var WebSocketShard = require('../node_modules/discord.js/src/client/websocket/WebSocketShard')
```

Comment:
Oh this is interesting. Let’s pull `WebSocketShard` from the library directly if it’s properly exported rather than from `node_modules`. That should address the TypeScript error, and provide better consistency across the codebase.

Reply:
https://old.discordjs.dev/#/docs/discord.js/v13/class/WebSocketShard